hack/build-images.sh

```diff
@@ -125,7 +122,7 @@ function image {
 ( image openshift/origin-docker-registry       images/dockerregistry ) &
 ( image openshift/origin-egress-router         images/router/egress ) &
```

Comment:
Shouldn't these be `build-image`?

Also, correctly written, `build-image` function shouldn't touch global state and there is no need for the subshells